.github/workflows/tests.yaml

```diff
@@ -19,6 +19,7 @@ jobs:
         python-version: 3.9
     - name: Install pip dependencies
       run: |
+        pip install git+https://github.com/kornia/kornia@master # required for mypy fixes
```

Comment:
Can you add a TODO to remove this once the next Kornia release comes out?

Reply:
Added TODO